src/sidebar/components/test/menu-keyboard-navigation-test.js

```diff
@@ -0,0 +1,119 @@
+import { mount } from 'enzyme';
+import { createElement } from 'preact';
+
+import MenuKeyboardNavigation from '../menu-keyboard-navigation';
+import { $imports } from '../menu-keyboard-navigation';
+
+import { checkAccessibility } from '../../../test-util/accessibility';
+import mockImportedComponents from '../../../test-util/mock-imported-components';
+
+describe('MenuKeyboardNavigation', () => {
+  let fakeCloseMenu;
+  let clock;
+
+  const createMenuItem = props => {
+    const container = document.createElement('div');
+    document.body.appendChild(container);
+    return mount(
+      <MenuKeyboardNavigation closeMenu={fakeCloseMenu} {...props}>
+        <button>Item 0</button>
+        <button role="menuitem">Item 1</button>
+        <button role="menuitem">Item 2</button>
+        <button role="menuitem">Item 3</button>
+      </MenuKeyboardNavigation>,
+      {
+        attachTo: container,
+      }
+    );
+  };
+
+  beforeEach(() => {
+    fakeCloseMenu = sinon.stub();
+    $imports.$mock(mockImportedComponents());
+  });
+
+  afterEach(() => {
+    $imports.$restore();
+  });
+
+  it('renders the provided class name', () => {
+    const wrapper = createMenuItem({ className: 'test' });
+    assert.isTrue(wrapper.find('.test').exists());
+  });
+
+  ['ArrowLeft', 'Escape'].forEach(key => {
+    it(`calls \`closeMenu\` callback when the ${key} is pressed`, () => {
+      const wrapper = createMenuItem({ visible: true }).find('div');
+      wrapper.simulate('keydown', { key });
+      assert.called(fakeCloseMenu);
+    });
+  });
+
+  // useFakeTimers does not work with checkAccessibility
+  // so wrap these tests in their own describe block
+  describe('fake timer', () => {
```

Comment:
```suggestion
  describe('keyboard navigation', () => {
```

"fake timer" is more of a detail of how these tests work rather than a description of what feature or scenario is being tested.